packages/iTwinUI-react/stories/core/Toasts.stories.tsx

```diff
@@ -98,6 +98,17 @@ export default {
     },
   },
   title: 'Core/Toasts',
+  decorators: [
+    (Story) => {
+      toaster.setSettings({
+        getRoot: (container) =>
+          ReactDOM.createRoot(
+            container ?? document.getElementsByTagName('div')[0],
+          ),
+      });
```

Comment:
Shouldn't we put this in the story itself instead of hiding in the decorator?

Reply:
no more story